src/js/components/Modal/Modal.tsx

```diff
@@ -58,8 +60,8 @@ const Modal: React.FC<Props> = ({
         <div className="Modal-header u-flex u-flexJustifyBetween">
           <div className="u-sizeFill">
             <h2 data-testid="modal-heading">{heading}</h2>
-          </div>
-          {showClose && (
+          </div>          
+          {showClose && closeButton ? closeButton : (
```

Comment:
Changed the approach here to link `showClose` and `closeButton` to create three states:
- `showClose` is false: no close icon
- `showClose` is true and `closeButton` is false:  uses default "x" icon
- `showClose` and `closeButton` are true:  uses custom close Button

I thought this worked better than making `showClose` and `closeButton` independent, since they shouldn't both be true.

@BillyFigueroa in the case of the wizard, Drew and I like having the "x" on the first three steps for an easy cancel (no save). So in the wizard code when `closeButton` is `null` the default `showClose` icon will show up.

Any concerns with this approach @justin-prince @BillyFigueroa @bradcypert ?

Reply:
@saralohr I like the 3 states